plugins/database/src/db_plugin.cpp

```diff
@@ -123,6 +116,98 @@ const std::string ZONE_PROP( "irods_zone_property" );
 
 static const auto intermediate_replica_status_str = std::to_string(INTERMEDIATE_REPLICA);
 
+namespace
+{
+    constexpr std::size_t grid_configuration_size = 2700;
+    struct auth_config
+    {
+        bool initialized = false;
+
+        bool password_extend_lifetime = true;
+
+        static constexpr rodsLong_t default_password_max_time = 1209600;
+        rodsLong_t password_max_time = default_password_max_time;
+        std::array<char, grid_configuration_size + 1> password_max_time_str{};
```

Comment:
This either needs to default to `default_password_max_time` as a string, or we don't want to keep this in the struct.